### ISSUE TITLE
Not able to specify assign_ipv6_address_on_creation for subnets

### ISSUE THREAD

**Issue by chrissam** [terraform-aws-modules/terraform-aws-vpc#185]
I don't find a way to specify `assign_ipv6_address_on_creation` for subnets in the vpc module, which is available as part of `aws_subnet` resource. 

**### Sample vpc config:**
```
module "vpc" {
  source = "terraform-aws-modules/vpc/aws"
	version = "1.46.0"

  name = "${var.vpc_name}"
  cidr = "${var.vpc_cidr_block}"

  azs             = "${var.availability_zones}"
  private_subnets = "${var.private_subnets}"
  public_subnets  = "${var.public_subnets}"

  enable_nat_gateway = true
  enable_vpn_gateway = true

  assign_generated_ipv6_cidr_block = true
	assign_ipv6_address_on_creation = true

  tags = {
    Terraform = "true"
  }
}
```

**Result:**
`Error: module "vpc": "assign_ipv6_address_on_creation" is not a valid argument`

**Comment by sharath-yml** [comment]
I had the same issue. As a workaround, I forked this project into: https://github.com/sharath-yml/terraform-aws-vpc and added this option for myself. I will spend more time on this later, and probably raise a PR on this after some time.

**Comment by antonbabenko** [comment]
[v2.10.0](https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/v2.10.0) has been released (see #317).
